frontend/helpers/graphql/queries/sidebar.ts

```diff
@@ -24,10 +24,11 @@ export const LOAD_SIDEBAR_1 = gql`
 `
 
 export const LOAD_SIDEBAR_2 = gql`
-    query LoadSidebar3($teamId: ID!) {
+    query LoadSidebar2($teamId: ID!) {
         team(id: $teamId) {
```

Comment:
The query for team API was recently changed to use slug instead of ID. 
Please double check its functionality, and update if needed.

Reply:
What is Sidebar2 differentiate from 1?
Pls use more descriptive variables. When we have new devs or you will get assigned into a team you can't tell which sidebar is this.